opentelemetry-instrumentation/tests/auto_instrumentation/test_sitecustomize.py

```diff
@@ -0,0 +1,47 @@
+from os import environ
+from unittest import TestCase
+from unittest.mock import patch
+
+
+class TestSiteCustomize(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.load_configurators = patch(
+            "opentelemetry.instrumentation.auto_instrumentation._load._load_configurators"
+        )
+        cls.load_instrumentors = patch(
+            "opentelemetry.instrumentation.auto_instrumentation._load._load_instrumentors"
+        )
+
+        cls.load_configurators.start()
+        cls.load_instrumentors.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.load_configurators.stop()
+        cls.load_instrumentors.stop()
+
+    def test_unset(self):
+        if environ.get("PYTHONPATH"):
+            del environ["PYTHONPATH"]  # enforce remove key/value from environ
+        from opentelemetry.instrumentation.auto_instrumentation import (  # noqa
```

Comment:
i think this will not have any effect if the sitecustomize module was already imported once previously (e.g. by a test from another module or by one of the tests below).
after the first import a module is put into `sys.modules` any subsequent imports will not run the module global code again.

Reply:
I think we can try an import only for a test without puting it into `sys.module` or remove the import at the end of the test.

Reply:
@lperdereau ping